### PR TITLE
Pcc01 das-dev

### PR DESCRIPTION
**Pull request by das-dev**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): [1]
Estimated time spent (hours): [1]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): []
Other feedback (what can we improve?): []

### CODE REVIEW

File: 01/das-dev/wordvalue.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+def load_words():
+    with open(DICTIONARY, 'r') as f:
+        return f.read().split()
+
+def calc_word_value(word):
+    return sum(LETTER_SCORES.get(letter.upper(), 0) for letter in word)
+
+def max_word_value(dictonary=None):
+    return max(dictonary or load_words(), key=calc_word_value)
```
Comment (pybites) [inline 01/das-dev/wordvalue.py]: nice use of `or`

File: 01/das-dev/wordvalue.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    with open(DICTIONARY, 'r') as f:
+        return f.read().split()
```
Comment (pybites) [inline 01/das-dev/wordvalue.py]: does this strip the newline of each line?

Comment (das-dev) [reply 01/das-dev/wordvalue.py]: > does this strip the newline of each line?

Is my code not correctly?

Comment (bbelderbos) [reply 01/das-dev/wordvalue.py]: It does work: 'print(f.read().split()[:10])' returns `['A', 'a', 'aa', 'aal', 'aalii', 'aam', 'Aani', 'aardvark', 'aardwolf', 'Aaron']`

Comment (bbelderbos) [reply 01/das-dev/wordvalue.py]: I am just used to writing `[line.strip() for line in f.readlines()]`

**Pull request merged**